packages/fxa-auth-server/lib/getRemoteAddressChain.js

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Any new file should be in TypeScript when possibble.

Reply:
Updated to use TS.